### DISCUSSION TITLE
500 internal server error when trying to login

### DISCUSSION THREAD

**Discussion by 911pcdoc** [magicbug/Cloudlog#973] [Q&A] [answered]
I built a new server (ubuntu 18.04 Nginx MySQL) and then clones the git to my desktop then FTPed it to the server completed the http://k4bum.radio/install no errors try now to login in with defaults and get 500 internal server error????

**Comment by magicbug** [comment, marked as answer]
Looks like a server configuration issue with nginx not much I can suggest bar check logs for errors

**Reply by 911pcdoc** [reply]
reloaded the server and went with apache all is working now
